cmd/cluster-dns-operator/main.go

```diff
@@ -39,6 +39,10 @@ func main() {
 	// namespace from config.
 	sdk.Watch("apps/v1", "DaemonSet", "openshift-dns", resyncPeriod)
 
+	releaseVersion := os.Getenv("RELEASE_VERSION")
+	if len(releaseVersion) == 0 {
+		logrus.Fatalf("RELEASE_VERSION environment variable is required")
```

Comment:
I don't think this should be a fatal error.  The description for https://github.com/openshift/cluster-image-registry-operator/pull/219 has the following:

> If no release version is set we reset versions to empty.
> 
> This is the model we will be implementing across all operators for 4.0.

So we can read it and store it in `operator.Config`, but we shouldn't treat its absence as fatal, and we should continue to conditionalize on non-zero length in `syncOperatorStatus`.